### PR TITLE
PSP-675 Add Tenant login background image

### PR DESCRIPTION
**Pull request by Fosol**
Login page background image is now controlled by the tenant configuration.

Implemented styled component theme provider to include tenant settings and global scss variables.

### CODE REVIEW

File: frontend/src/index.tsx [hunk 1]
```diff
@@ -19,30 +19,40 @@ import { BrowserRouter as Router } from 'react-router-dom';
 import EmptyLayout from 'layouts/EmptyLayout';
 import LoginLoading from 'features/account/LoginLoading';
 import { store } from 'store/store';
-import { TenantProvider } from 'tenants';
+import { TenantConsumer, TenantProvider } from 'tenants';
+import { ThemeProvider } from 'styled-components';
+
+// eslint-disable-next-line import/no-webpack-loader-syntax
+const css = require('sass-extract-loader?{"plugins": ["sass-extract-js"]}!./_variables.scss');
```
Comment (Fosol) [inline frontend/src/index.tsx:26]: I would love a better way to do this. 
This extracts scss variables and provides a way to include them into the styled component theme.

File: frontend/src/index.tsx [hunk 2]
```diff
@@ -19,30 +19,40 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
-          <AuthStateContextProvider>
-            <Router>
-              <App />
-            </Router>
-          </AuthStateContextProvider>
-        </Provider>
-      </ReactKeycloakProvider>
+      <TenantConsumer>
+        {({ tenant }) => (
+          <ThemeProvider theme={{ tenant, css }}>
```
Comment (Fosol) [inline frontend/src/index.tsx:35]: Add the scss global vars and the tenant config into the styled component theme.

**Review by jdtoombs — APPROVED** [review]

File: frontend/src/layouts/__snapshots__/Header.test.tsx.snap [hunk 3]
```diff
@@ -4,7 +4,7 @@ exports[`Tenant Header Header CITZ background 1`] = `
 .c0 {
   background-color: #003366;
   border: none;
-  border-bottom: 2px solid $accent-color;
+  border-bottom: 2px solid;
```
Comment (devinleighsmith) [inline frontend/src/layouts/__snapshots__/Header.test.tsx.snap:7]: Hmm, are we sure the props.them.css.accentColor is working? Maybe it just isn't working in the snapshots? I would expect to see the color show up here.

**Review by devinleighsmith — APPROVED** [review]

File: frontend/src/layouts/__snapshots__/Header.test.tsx.snap (hunk 3 above)
Comment (Fosol) [reply frontend/src/layouts/__snapshots__/Header.test.tsx.snap:7]: The snapshot would require mocking the global scss variables, which I don't believe I have

**Pull request merged**